pkgs/by-name/re/resticprofile/package.nix

```diff
@@ -9,15 +10,19 @@
   resticprofile,
 }:
 
-buildGoModule rec {
+let
+  # can be removed when the default go version is at least 1.23
+  buildGoModule' = buildGoModule.override { go = go_1_23; };
```

Comment:
Why wasn't buildGo123Module used instead?

Reply:
Well, I did not know it existed.